features/basics/schema_metadata.feature

```diff
@@ -60,81 +59,107 @@ Feature: Schema Metadata
-      end
+      index = cluster.keyspace('simplex').table('test_table').index('ind1')
+      puts index.to_cql
+
+      puts ""
+      puts "Name: #{index.name}"
+      puts "Table name: #{index.table.name}"
+      puts "Kind: #{index.kind}"
+      puts "Target: #{index.target}"
       """
     When it is executed
     Then its output should contain:
       """cql
-      CREATE INDEX "ind1" ON simplex.test_table ("f2");
+      CREATE INDEX "ind1" ON simplex.test_table (b);
+
+      Name: ind1
+      Table name: test_table
+      Kind: composites
+      Target: b
       """
 
   @cassandra-version-specific @cassandra-version-3.0
-  Scenario: Getting index metadata on 3.0
+  Scenario: Getting index metadata on full collections
     Given the following schema:
       """cql
       CREATE KEYSPACE simplex WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
-      CREATE TABLE simplex.test_table (f1 int primary key, f2 int);
-      CREATE INDEX ind1 ON simplex.test_table (f2);
+      CREATE TABLE simplex.test_table (a int PRIMARY KEY, b frozen<map<text, text>>);
       """
     And the following example:
       """ruby
       require 'cassandra'
 
       cluster = Cassandra.cluster
+      session = cluster.connect
 
-      cluster.keyspace('simplex').table('test_table').each_index do |index|
-        puts index.to_cql
-      end
+      session.execute("CREATE INDEX ind1 ON simplex.test_table (full(b))")
+      sleep(2)
+      index = cluster.keyspace('simplex').table('test_table').index('ind1')
+      puts index.to_cql
+
+      puts ""
+      puts "Name: #{index.name}"
+      puts "Table name: #{index.table.name}"
+      puts "Kind: #{index.kind}"
+      puts "Target: #{index.target}"
       """
     When it is executed
     Then its output should contain:
       """cql
-      CREATE INDEX "ind1" ON simplex.test_table (f2);
+      CREATE INDEX "ind1" ON simplex.test_table (full(b));
+
+      Name: ind1
+      Table name: test_table
+      Kind: composites
+      Target: full(b)
       """
 
   @cassandra-version-specific @cassandra-version-3.0
-  Scenario: Getting materialized view metadata
+  Scenario: Getting multiple index metadata on same column
     Given the following schema:
       """cql
       CREATE KEYSPACE simplex WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
-      CREATE TABLE simplex.test_table (f1 int PRIMARY KEY, f2 int, f3 int) ;
-      CREATE MATERIALIZED VIEW simplex.test_view AS
-       SELECT f1, f2 FROM simplex.test_table WHERE f2 IS NOT NULL
-       PRIMARY KEY (f1, f2);
+      CREATE TABLE simplex.test_table (a int PRIMARY KEY, b map<text, text>);
       """
     And the following example:
       """ruby
       require 'cassandra'
 
       cluster = Cassandra.cluster
+      session = cluster.connect
 
-      view = cluster.keyspace('simplex').materialized_view('test_view')
-      puts view.to_cql
+      session.execute("CREATE INDEX ind1 ON simplex.test_table (keys(b))")
```

Comment:
Same here; if index creation moves into 'given...' section, can we avoid the sleep below?